src/client/js/components/Admin/SlackIntegration/CustomBotWithoutProxySettings.jsx

```diff
@@ -29,8 +27,13 @@ const CustomBotWithoutProxySettings = (props) => {
     }
   };
 
+  const fetchSiteName = () => {
+    const siteName = appContainer.getConfig().crowi.title;
```

Comment:
```suggestion
    const siteName = appContainer.config.title;
```
getConfig を実行しなくても上記のコードでとれる気がします

Reply:
appContainer.config.title をログに出してみましたが、undefined となってしまいました。

Reply:
↑ appContainer.config.crowi.title でサイト名を取得できました！